### ISSUE TITLE
Cannot load Fonts if etherpad is behind proxy with subfolder

### ISSUE THREAD

**Issue by trinhpham** [ether/etherpad-lite#3986]
I used the latest docker image of etherpad (tag 1.8.3), run it behind an nginx reverse proxy at subfolder (eg. mydomain.com/etherpad), and observed that the site was failed at trying to get some fonts at `mydomain.com/static/fonts/` instead of `mydomain.com/etherpad/static/fonts/`.
I can't make a workaround to proxy the static folder since my main site has that folder too.

My nginx config:
```
# Etherpad-lite
location /etherpad/ {
    proxy_http_version 1.1;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection 'upgrade';
    proxy_set_header Host $host;
    proxy_cache_bypass $http_upgrade;
    proxy_set_header X-Forwarded-For $remote_addr;
    proxy_buffering off;
    proxy_pass http://etherpad-docker:9001/;
}
```

Any idea to fix this?

**Comment by JohnMcLear** [comment]
Duplicate of https://github.com/ether/etherpad-lite/issues/3956

**Comment by JohnMcLear** [comment]
No fix yet, it's #4 on my list so hopefully someone can get to it before me.
